Traceback (most recent call last):
  File "main.py", line 76, in <module>
    agent.qnetwork_local.load_state_dict(torch.load('checkpoint.pth'))
  File "/gpfsdata/home/xiyuezhu/miniconda3/lib/python3.7/site-packages/torch/nn/modules/module.py", line 847, in load_state_dict
    self.__class__.__name__, "\n\t".join(error_msgs)))
RuntimeError: Error(s) in loading state_dict for QNetwork:
	size mismatch for fc1.weight: copying a param with shape torch.Size([64, 8]) from checkpoint, the shape in current model is torch.Size([256, 128]).
	size mismatch for fc1.bias: copying a param with shape torch.Size([64]) from checkpoint, the shape in current model is torch.Size([256]).
	size mismatch for fc2.weight: copying a param with shape torch.Size([64, 64]) from checkpoint, the shape in current model is torch.Size([256, 256]).
	size mismatch for fc2.bias: copying a param with shape torch.Size([64]) from checkpoint, the shape in current model is torch.Size([256]).
	size mismatch for fc3.weight: copying a param with shape torch.Size([4, 64]) from checkpoint, the shape in current model is torch.Size([4, 256]).
